airflow/www/api/v0/endpoints.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+
+from airflow import models
+from airflow import configuration as conf
+
+from flask import Blueprint, jsonify
+
+apiv0 = Blueprint('apiv0', __name__)
+
+dagbag = models.DagBag(os.path.expanduser(conf.get('core', 'DAGS_FOLDER')))
```

Comment:
please use the one available from views.py . Neither ideal, but prevents loading the DagFolder again. If we update the dagbag there it will also be reflected here otherwise you need to do that by hand.